library/tasks/SmartSeq2PlateAggregation.wdl

```diff
@@ -0,0 +1,114 @@
+task AggregateDataMatrix {
+  Array[File] filename_array
+  String col_name
+  String output_name
+  String docker = "quay.io/humancellatlas/secondary-analysis-ss2-plate-aggregation:0.0.1"
+
+  meta {
+    description: "aggregate output count matrix"
+  }
+
+  command {
+    set -e
+    python /tools/MergeDataMatrix.py -f ${sep=' ' filename_array}  -t ${col_name} -o ${output_name}
+  }
+
+  output{
+    File aggregated_result = "${output_name}"
+  }
+
+  runtime {
+    docker: docker
+    memory: "2 GB"
+    disks: "local-disk 10 HDD"
+    cpu: 1
+    preemptible: 5
+    maxRetries: 1
+  }
+}
+
+task AggregateQCMetrics {
+  Array[File] metric_files
+  String output_name
+  String docker = "quay.io/humancellatlas/secondary-analysis-ss2-plate-aggregation:0.0.1"
+  String run_type
+
+  meta {
+    description: "aggregate count data"
```

Comment:
Interesting, I haven’t thought of QC metrics as count data before, maybe there’s another way to word this?